client_cli/src/main.rs

```diff
@@ -158,19 +168,34 @@ fn main() -> Result<()> {
                 .as_ref(),
         )?
     };
+
     let Configuration(config) = config;
-    println!(
-        "User: {}@{}",
-        config.account_id.name, config.account_id.domain_id
-    );
-    #[cfg(debug_assertions)]
-    eprintln!(
-        "{}",
-        &json5::to_string(&config).wrap_err("Failed to serialize configuration.")?
-    );
-    #[cfg(not(debug_assertions))]
-    eprintln!("This is a release build, debug information omitted from messages");
-    subcommand.run(&config)?;
+
+    if verbose {
+        eprintln!(
+            "User: {}@{}",
+            config.account_id.name, config.account_id.domain_id
```

Comment:
Comment on why this is not `account_id`'s `core::fmt::Display`

Reply:
No reason, can be changed. Probably leftover from when AccountId didn't implement Display or something